eureka2-client/src/main/java/com/netflix/eureka2/client/Eureka.java

```diff
@@ -17,9 +17,9 @@
 package com.netflix.eureka2.client;
 
 import com.netflix.eureka2.client.resolver.ServerResolver;
-import com.netflix.eureka2.client.resolver.ServerResolver.Server;
 import com.netflix.eureka2.client.resolver.ServerResolvers;
 import com.netflix.eureka2.transport.EurekaTransports;
+import com.netflix.eureka2.utils.Server;
```

Comment:
As ```Server``` is part of public API, it should not reside in utils package. Can we move it to some other place (com.netflix.eureka2?)